pkg/prometheus/rules.go

```diff
@@ -207,6 +200,6 @@ func checksumRuleFiles(files map[string]string) string {
 	return fmt.Sprintf("%x", sha256.Sum256([]byte(sum)))
 }
 
-func prometheusRuleFilesConfigMapName(prometheusName string) string {
+func prometheusRuleConfigMapName(prometheusName string) string {
 	return "prometheus-" + prometheusName + "-rulefiles"
```

Comment:
Do we want to keep `-rulefiles` here? Could be just `rules` instead?

Reply:
I am also fine leaving it like it is, keeping it as an internal implementation detail.

Reply:
I chose to stick with `-rulefiles` because it's very common (even in kube-prometheus this is the case) that people have `<prometheus-name>-rules` as their configmap name today, in which case it will get overridden, so I chose not to and in this case `-rulefiles` is ok to be specific I think.